opentelemetry-appender-tracing/src/layer.rs

```diff
@@ -90,20 +94,31 @@ where
 
 impl<S, P, L> Layer<S> for OpenTelemetryTracingBridge<P, L>
 where
-    S: tracing::Subscriber,
+    S: Subscriber + for<'a> LookupSpan<'a>,
     P: LoggerProvider<Logger = L> + Send + Sync + 'static,
     L: Logger + Send + Sync + 'static,
 {
-    fn on_event(
-        &self,
-        event: &tracing::Event<'_>,
-        _ctx: tracing_subscriber::layer::Context<'_, S>,
-    ) {
+    fn on_event(&self, event: &tracing::Event<'_>, ctx: tracing_subscriber::layer::Context<'_, S>) {
         let meta = event.metadata();
         let mut log_record: LogRecord = LogRecord::default();
         log_record.severity_number = Some(severity_of_level(meta.level()));
         log_record.severity_text = Some(meta.level().to_string().into());
 
+        // Extract the trace_id & span_id from the opentelemetry extension.
```

Comment:
https://github.com/open-telemetry/opentelemetry-rust/issues/1378 - Looks like this PR is attempting to solving the issue?

Reply:
That's correct! I didn't notice the original issue, I'll update the PR description to include it.